frontend/www/js/omegaup/components/contest/Editv2.vue

```diff
@@ -172,6 +172,25 @@
         ></omegaup-contest-groups>
       </div>
     </div>
+    <div v-if="showTab === 'admins'" class="tab-pane active">
+      <omegaup-contest-admins
+        :admins="admins"
+        :has-parent-component="true"
+        @emit-add-admin="(username) => $emit('add-admin', username)"
+        @emit-remove-admin="(username) => $emit('remove-admin', username)"
```

Comment:
```suggestion
        @add-admin="(username) => $emit('add-admin', username)"
        @remove-admin="(username) => $emit('remove-admin', username)"
```

y acá se simplifica la llamada! igual abajo.